packages/itwinui-css/src/tile/tile.scss

```diff
@@ -353,19 +352,24 @@ $thumbnail-selector: ':is(.iui-thumbnail-icon, .iui-tile-thumbnail-picture)';
     justify-self: end;
   }
 
-  .iui-tile-thumbnail-picture ~ & {
-    @include mixins.iui-blur($opacity: 5);
-    --_iui-button-icon-fill: var(--iui-color-white);
+  .iui-button[data-iui-variant='borderless'] {
```

Comment:
It might be better to create a new class for this instead of referencing iui-button selectors here.

Reply:
Also this needs a changeset.

Reply:
> It might be better to create a new class for this instead of referencing iui-button selectors here.

@r100-stack said the user can place anything in this slot.  So we would need the user to pass this new class for it to work correctly, no?

Reply:
Technically yes, the user can place anything, but it's mainly designed to work with `Tile.IconButton`, where we can add this class automatically.

Reply:
If you mean like `> *`, yes that would work for most cases, but not for this:

```jsx
<Tile.QuickAction>
  <div>
    <Tile.IconButton>…</Tile.IconButton>
  </div>
</Tile.QuickAction>
```

It could also have unintended consequences for this scenario (probably rare):

```jsx
<Tile.QuickAction>
  <Tile.IconButton>…</Tile.IconButton>
  <div>…</div>
</Tile.QuickAction>
```

I think it makes sense to have a dedicated class for `Tile.IconButton`, since it has its own distinct look and feel. `Tile.QuickAction` and `Tile.TypeIndicator` should only handle positioning.

One thing to consider, however, is that this could be a potential breaking change. Maybe that's ok? We already require users to [change their usage](https://github.com/iTwin/iTwinUI/wiki/iTwinUI-react-v3-migration-guide#tile) going from v2→v3.

Reply:
Ahh, I see.  I've added a new `.iui-tile-thumbnail-button` and updated both the CSS demos and React.

Reply:
Should we move the Tile changes into a separate PR? I think it warrants additional discussion.

Reply:
Ok, working on that when I get back to my desk tonight.

Reply:
Moved tile content into #2101.  Images are failing now on this PR, I didn't feel comfortable approving test images knowing they are incorrect, unsure how to proceed without making some sort of change within tile.

Reply:
This PR should preferably have the minimum changes necessary for making the images match the `main` branch. Is it possible to do that without large changes in `tile.scss`?

Reply:
https://github.com/iTwin/iTwinUI/pull/2088/commits/73cb4ed3c00fc2d41c3149c4a756a9706ab14c73 seems to get the job done for now.